app/src/main/kotlin/com/infinity/foodstandards/ui/hygieneRating/HygieneRatingsCompose.kt

```diff
@@ -0,0 +1,80 @@
+package com.infinity.foodstandards.ui.hygieneRating
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.infinity.foodstandards.R
+import com.infinity.foodstandards.model.Establishment
+
+@Composable
+fun AuthorityDetailDescription() {
+    AuthorityName()
+}
+
+@Preview
+@Composable
+fun AuthorityNamePreview() {
+    AuthorityName()
+}
+
+@Composable
+fun RowScope.TableCell(
+    text: String,
+    weight: Float
+) {
+    Text(
+        text = text,
+        Modifier
+            .border(1.dp, Color.Black)
+            .weight(weight)
+            .padding(8.dp)
+    )
+}
+
+@Composable
+private fun AuthorityName() {
```

Comment:
I think we should probably hoist the column percentage values out of the component here to make it usable during the test, even simply providing these as parameters would be better (or as an object to provide as part of the view state) e.g.

```
@Composable
fun AuthorityDetailTable(val fiveStarRating: String, val fourStarRating: String, val threeStarRating: String, val twoStarRating: String, val oneStarRating: String, val exemptRating: String) { ... }
```

We can then get rid of the `AuthorityDetailDescription` composable wrapper and provide a preview with the 16.6% values 👍

Reply:
I don't think we need the `AuthorityDetailDescription` composable? We could just emit that directly from the Fragment, but the update here is all good 👍